README.md

```diff
@@ -469,4 +470,9 @@ Modules in plugin/modules
 * [ipauser](README-user.md)
 * [ipavault](README-vault.md)
 
+Inventory plugins in plugin/inventory
+=====================================
+
+* [freeipa](README-inventory-plugin-freeipa.md)
+
```

Comment:
The last line is related to modules and not the inventory plugin.

Either this section is moved after it, or a new section "Writing new modules" is added here.